radix-engine-tests/tests/bootstrap.rs

```diff
@@ -380,27 +379,24 @@ fn test_genesis_time() {
         .bootstrap_with_genesis_data(
             vec![],
             1u64,
-            dummy_epoch_manager_configuration(),
+            dummy_consensus_manager_configuration(),
             123 * 60 * 1000 + 22, // 123 full minutes + 22 ms (which should be rounded down)
         )
         .unwrap();
 
-    let current_time_substate = substate_db
-        .get_mapped::<SpreadPrefixKeyMapper, ClockSubstate>(
-            CLOCK.as_node_id(),
+    let current_time_rounded_to_minutes_ms = substate_db
+        .get_mapped::<SpreadPrefixKeyMapper, i64>(
```

Comment:
I have a slight preference for this being an explicit named type - even if it's just a `type` alias.